content/tutorial/tutorial.md

```diff
@@ -54,7 +54,7 @@ If you want to do it, here are the steps to follow:
 
 1. Make sure you have a recent version of [Node.js](https://nodejs.org/en/) installed.
 2. Follow the [installation instructions](/docs/installation.html#creating-a-new-application) to create a new project.
-3. Delete all files in the `src/` folder of the new project.
+3. Delete all files inside the `src/` folder of the new project (don't exclude the folder, just the files).
```

Comment:
> (don't exclude the folder, just the files).

I'm not sure the wording for this is clear. How do you feel about:

> don't delete the folder, just its contents (eg `rm -f src/*`)

Reply:
nice, I'm update this.